src/Hazelcast.Net.Testing/MultiMembersRemoteTestBase.cs

```diff
@@ -59,6 +59,20 @@ protected async Task RemoveMember(Guid memberId)
             }
         }
 
+        /// <summary>
+        /// Shutdowns a member from the cluster.
+        /// </summary>
+        /// <param name="memberId">The identifier of the member to remove.</param>
+        /// <returns>A task that will complete when the member has been shutdown.</returns>
+        protected async Task ShutdownMember(Guid memberId)
```

Comment:
In https://github.com/hazelcast/hazelcast-csharp-client/pull/614/files this is changed to RemoveMember

Reply:
That is newer than the ported commit. So that is why they are different. I didn't touch the code as much as possible.